packages/vat-data/src/environment-options.js

```diff
@@ -0,0 +1,135 @@
+// @ts-check
+
+// NOTE This module is essentially a duplicate of a module by the same name
+// in the endo repository. It is copied here *temporarily* until the one in
+// endo exports bindings we can import.
```

Comment:
Would using `patch-package` also have worked?

Reply:
In theory, but we don't yet have such a separate package at the endo level so it's not clear what the patch would say.

In any case, recent experience makes me shy away from introducing new agoric-sdk dependencies on a patch to endo.